### DISCUSSION TITLE
Which language is recommended to add in this study material repository?

### DISCUSSION THREAD

**Discussion by rkshaon** [rkshaon/software-engineering-preparation#43] [Q&A] [answered]
This study material repository has Python as Programming language, PostgreSQL, Redis as Database language. Is there any other language recommended to add here?

**Comment by RemonAhammad** [comment, marked as answer]
Great foundation! Consider adding:

Front-end languages (HTML, CSS, JavaScript) to showcase how the Python backend connects to user interfaces.
Testing framework (e.g., pytest) to emphasize code quality and maintainability. These additions broaden the repository's appeal to full-stack developers.
